lapce-data/src/data.rs

```diff
@@ -133,7 +136,21 @@ impl LapceData {
         let latest_release = Arc::new(None);
 
         let dirs: Vec<&PathBuf> = paths.iter().filter(|p| p.is_dir()).collect();
-        let files: Vec<&PathBuf> = paths.iter().filter(|p| p.is_file()).collect();
+
+        // let (files, line_columns) = Self::parse_paths_line_column(paths.clone());
+        
+        // parse paths into (file, line, column) tuples
+        let parsed_paths:Vec<Option<(PathBuf, Option<LineCol>)>> = paths.iter().map(|p| Self::parse_path_line_column(p)).collect();
```

Comment:
Must be
```rust
let parsed_paths: Vec<(PathBuf, Option<LineCol>)> = paths.iter().map(|p| Self::parse_path_line_column(p)).collect();
```

Reply:
Or `let parsed_paths: Vec<(PathBuf, Option<LineCol>)> = paths.iter().map(|p| Self::parse_path_line_column(p)).collect()`? See below.

Reply:
can't remember exactly if collect can handle this, if not just add `filter_map(|p| p)` before `collect `